### ISSUE TITLE
CSV export has unreadable geometry info

### ISSUE THREAD

**Issue by saleiva** [CartoDB/cartodb#9654]
we should have more readable format, for example WKT or separate lat/lng fields.

cc/ @rafatower

**Comment by rafatower** [comment]
The export is actually implemented in SQL API cc/ @rochoa 

Regarding the output format, I'm not strongly against but that is something debatable. WKT can be ok for points but not so for polygons or other data types. I guess it has a bigger overhead in terms of both size/bandwidth and db load during export operation.

**Comment by rochoa** [comment]
Readable format? What does that mean? When do you expect to have that "readable format"? When the user uses the export options from the builder UI?

**Comment by saleiva** [comment]
always I believe. Think on the use case where people uses CARTO to host data that will be exported by the visitors of their site.

**Comment by rochoa** [comment]
I don't agree. If I'm using the SQL API I want to control the output format. If you decide to change the format from, let's say, WKB to WKT or lon+lat I won't be able to control it. If it's something we want to do when you click an export button I'm OK. But, why if there is a name clash?

**Comment by saleiva** [comment]
Im fine with that, with always I meant always where not experts are involved.
